news/fix_to_openmm_topology_bonds.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* <news item>
+
+**Deprecated:**
+
+* <news item>
+
+**Removed:**
+
+* <news item>
+
+**Fixed:**
+
+* Fixed behavior of `ProteinComponent.to_openmm_topology` to correctly assign `int` or `None` types to bond order for OpenMM, instead of objects. It now also adds some bond type information as objects, if detected.
```

Comment:
```suggestion
* Fixed behavior of `ProteinComponent.to_openmm_topology` to correctly assign `int` or `None` types to bond order for OpenMM, instead of objects. It now also adds some bond type information as objects, if detected. (#501)
```

Reply:
On a second thought, I actually think the part that adds some bond type information is a _change_ rather than a _fix_. Does that part make more sense to add it as a change?

Reply:
Hmm, I guess it is kinda a philosophical question but since the thing we were doing was causing an error, I would call it a fix. We had to change things to fix it, but I would argue that they way we were doing was incorrect.